master/docs/developer/mq.rst

```diff
@@ -105,11 +105,50 @@ Simple
 
 .. py:module:: buildbot.mq.simple
 
-.. py:class:: SimpleConnector
+.. py:class:: SimpleMQ
 
     The :py:class:`SimpleMQ` class implements a local equivalent of a message-queueing server.
     It is intended for Buildbot installations with only one master.
 
+Wamp
+....
+
+.. py:module:: buildbot.mq.wamp
+
+.. py:class:: WampMQ
+
+    The :py:class:`WampMQ` class implements message-queueing using a wamp router.
+    This class translates the semantics of the buildbot mq api to the semantics of the wamp messaging system.
+    A notable difference is that autobahn API does not send the original topic to the callback, so we need to include it inside the data part, for the case of the wildcard subscriber.
+    Example message that is sent via wamp is:
```

Comment:
You can get the topic passed to your callback via an optional kwarg.

You can get this kwarg passed to your callback by specifiying ``details_arg='details'`` in a SubscribeOptions -- this would cause a "details=" kwarg to be passed to your callback (which will be an ``EventDetails`` instance, which has a ``.topic`` member)